iroha_actor/src/deadlock.rs

```diff
@@ -185,9 +185,9 @@ impl Handler<RemoveEdge> for DeadlockActor {
 /// TODO: After switching to tokio move to using once cell and
 /// force initing of actor beforehand in async api.
 static DEADLOCK_ACTOR: Lazy<Addr<DeadlockActor>> = Lazy::new(|| {
-    let actor = DeadlockActor::init_default();
-    let address = actor.address().clone();
-    let _result = task::spawn(async move { actor.start().await });
+    let actor = DeadlockActor::preinit_default();
```

Comment:
If we don't have `init` then why call method `preinit`?

Reply:
Because someone might missuse it. I think that generally you need to know for sure that you want to use it. That is why i thought that it might be a good idea to name it this way.

Reply:
init name implicitly requires you to run it before start, but preinit says that you might run it

Reply:
I think just init is ok, the interface is pretty easy - init().start() or just .start()

Reply:
Idk, start is semantically init in our calls. So that is why thing which happends before start is actually pre init.